awx/main/management/commands/register_default_execution_environments.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Ansible, Inc.
+# All Rights Reserved
+import sys
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from awx.main.models import CredentialType, Credential, ExecutionEnvironment
+
+
+class Command(BaseCommand):
+    """Create default execution environments, intended for new installs"""
+
+    help = """
+    Creates or updates the execution environments set in settings.DEFAULT_EXECUTION_ENVIRONMENTS if they are not yet created.
+    Optionally provide authentication details to create or update a container registry credential that will be set on all of these default execution environments.
+    Note that settings.DEFAULT_EXECUTION_ENVIRONMENTS is and ordered list, the first in the list will be used for project updates and system jobs.
+    """
```

Comment:
@shanemcd @jbradberry I updated this help text, what do you think?

Reply:
"... is an ordered list"